docs/quickstart.md

```diff
@@ -379,3 +379,32 @@ value to be more or less strict:
 ```python
 >>> httpx.get('https://github.com/', timeout=0.001)
 ```
+
+## .netrc Support
```

Comment:
What do you think about moving this to the advanced section? Do we have a section just for all the different authentication methods?

I'd rather keep the quick start section brief and then have an extensive advanced usage / API section.